run.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+pip3 install -r requirements.txt
+cp application.py app.py
+apt install -y python3-flask
```

Comment:
Isnt flask a dependency installed by requirements.txt?

Reply:
it seems like in requirements there is Flask but when i tried to run it on the machine i had to pip3 install python3-flask in order for it to work. ill just add it as a dependency in requirements.txt